src/material/slide-toggle/BUILD.bazel

```diff
@@ -29,6 +29,7 @@ ng_module(
         "@npm//@angular/core",
         "@npm//@angular/forms",
         "@npm//@angular/platform-browser",
+        "@npm//@material/ripple",
```

Comment:
This seems to be added by mistake. Nothing in the standard Angular Material implementation uses `@material/ripple`.